app/GraphQL/Mutations/Login.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\User;
+use GraphQL\Error\Error;
+use Illuminate\Support\Facades\Auth;
+
+class Login
+{
+    /**
+     * @param  null  $_
+     * @param  array<string, mixed>  $args
+     */
+    public function __invoke($_, array $args): User
+    {
+        $guard = Auth::guard(config('sanctum.guard', 'web'));
```

Comment:
config('sanctum.guard', 'web') returns an array I believe so you would probably have to put config('sanctum.guard', 'web')[0]

Reply:
Thank you for the hint, fixed it.